hybridse/src/udf/udf_test.cc

```diff
@@ -664,6 +664,86 @@ TEST_F(ExternUdfTest, TestCompoundTypedExternalCall) {
         &library, "make_tuple", TupleResT(1, nullptr, 3), 1, nullptr, 3);
 }
 
+TEST_F(ExternUdfTest, LikeMatchTest) {
```

Comment:
Can we perform `LIKE` on Chinese characters or other Unicode chars?
If so, add some related unit tests and YAML cases.
If non-support currently, add some comments and documentation. And throw an error during converting ASTNode or expression evaluation.

Reply:
I'm afraid not, see #808 , the string going to compare don't have locale info, and the simple u8 char compare is performed

Reply:
It's okay. Inform users in function documentation and our language guide.